web/modules/custom/solution/config/install/core.base_field_override.rdf_entity.solution.label.yml

```diff
@@ -14,4 +14,4 @@ translatable: true
 default_value: { }
 default_value_callback: ''
 settings: { }
-field_type: string
+field_type: string
```

Comment:
There is no newline, this is the most "invisible" of the coding standards :) No worries, I'll fix it.